app/routes/frecuentes.tsx

```diff
@@ -0,0 +1,25 @@
+import {Link} from '@remix-run/react';
+
+import {NavBar, NavBarLink} from '~/components/organisms/navbar';
+import {RegularsHero} from '~/components/organisms/regulars-hero/regulars-hero.component';
+import {Footer} from '~/components/templates/footer';
+import configData from '~/config.json';
+
+// @ts-ignore
```

Comment:
Why is `@ts-ignore` needed here?

Reply:
The usage of this `@ts-ignore` comes from an early version of the `NavBar` component created by Tony. Remix Run's `Link`s are incompatible with Storybook, and so this workaround was made. I tried creating a type for it on the NavBar's final version and it just seemed too difficult for me, honestly.